src/input/ApfConvertWrapper.h

```diff
@@ -0,0 +1,31 @@
+#ifndef APF_CONVERT_WRAPPER_H
+#define APF_CONVERT_WRAPPER_H
+
+#include <apfConvert.h>
+#include <type_traits>
+
+// This is a small fix, because some time after PUMI (i.e. SCOREC/core) v2.2.7,
```

Comment:
Do I understand correctly that the problem is actually that pumi is at version v2.2.7 
https://github.com/SCOREC/core/blob/master/CMakeLists.txt#L9
Once the version is iterated you will be able to use
https://github.com/SCOREC/core/blob/master/pumi/pumi_version.h.in#L3

Reply:
The problem is that after the tag v2.2.7, some index type was changed from `int` to `long`, and thus PUMgen did not compile anymore. If we switch to the newer type, then all older versions of SCOREC will not work anymore with PUMgen. I.e. the `ApfConvertWrapper.h` is a small hack to make it work with all versions of SCOREC.

Unfortunately, the `pumi_version.h` header will not help us here (even when reaching some newer version), as we need the type at compile-time—but `pumi_version.h` only provides it at run-time.